### DISCUSSION TITLE
Wallet - Bump Fee / RBF Transactions

### DISCUSSION THREAD

**Discussion by astupidmoose** [btcpayserver/btcpayserver#2082] [Ideas / Feature Requests]
While BTCPay supports the ability to send RBF fees, there is no ability to resend or "bump" the fee on these transactions. This would require using an alternative wallet such as Electrum. 

I suggest implementing a method to select an outbound transaction and being able to bump the fee and then rebroadcast.

**Comment by pavlenex** [comment]
Added in 2.1.0 https://blog.btcpayserver.org/btcpay-server-2-1-0/#new-fee-acceleration-ui-and-rbf-support
